Update PR template (#331)
NethermindEth/juno: .github/pull_request_template.md

Fixes | Closes | Resolves #

## Description

> _Explanation about why the PR is necessary_

Reviewer: ```suggestion
_Explanation about why the PR is necessary._
```

## Changes:

- Change 1
- Change 2

## Types of changes

> _Leave only the ones that describe your changes, remove the rest. Can remove this line too_

Reviewer: ```suggestion
_Leave only items that describe your changes, remove the rest. Remove this line too._
```

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Update
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no api changes)
- Build related changes
- Other (please describe): 

> ## Testing

Reviewer 2: Let's add a space after the heading.

```suggestion
## Testing

```

**Requires testing**: Yes/No

> **Did you write tests??**: Yes/No

Reviewer 2: This renders as a single line. Perhaps a space in between will fix it?

Author: Done -> https://github.com/NethermindEth/juno/pull/331/commits/3c9da69405a173a192731bcbcafb856b8defd659
